_includes/generated-icons.html

```diff
@@ -37,6 +37,10 @@
     
   
     
+    <li class="icon-list-item"><h5>.icon-arrow-right-circle-24</h5><div class="icon-arrow-right-circle-24" style="width: 24px; height: 24px;" ></div></li>
```

Comment:
Mainitaining consistency would be nice here. I think this line is missing one leading space.
The same applies to dist/assets/icons/preview.html  L40